source/FreeRTOS_DHCPv6.c

```diff
@@ -1102,6 +975,7 @@ static void prvSendDHCPMessage( NetworkEndPoint_t * pxEndPoint )
                     break;
 
                 case eWaitingOffer:
+                case eLeasedAddress:

```

Comment:
When the address from DHCPv6 is timeout, we need to send DHCPv6 request again for re-new.